contribs/docker/wazo-auth-mock/wazo-auth-mock.py

```diff
@@ -310,8 +310,10 @@ def token_post():
         if password == valid_credentials[username]['password']:
             token = valid_credentials[username]['token']
             if token in valid_tokens:
-                return jsonify({'data': {'auth_id': valid_tokens[token]['auth_id'],
-                                         'token': token}})
+                result = dict(valid_tokens[token])
+                result.setdefault('xivo_user_uuid', result['auth_id'])
```

Comment:
No code should check this value, so I would avoid it if possible. In the best case, we will discover all deprecated code :)

Reply:
I just copy/paste GET :), I removed it!